libraries/Filter/NotchFilter.h

```diff
@@ -29,12 +29,17 @@
 template <class T>
 class NotchFilter {
 public:
+    template <class U> friend class HarmonicNotchFilter;
```

Comment:
Yes. I went back on forth on making HarmonicNotch inherit from NotchFilter, but there is no need other than to make it a friend, also even with inheritance you still need a friend so that the HarmonicNotch can initialize the filter bank.